app/views/admin/products_v3/_filters.html.haml

```diff
@@ -2,9 +2,10 @@
   .query
     .search-input
       = text_field_tag :search_term, search_term, placeholder: t('.search_products')
-  .producers
-    .label= t('.producers.label')
-    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers')
+  - if producer_options.length > 1
```

Comment:
Maybe this could be simpler:
```suggestion
  - if producer_options.present?
```
or is this better (I think means the same thing):
```suggestion
  - if producer_options.any?
```

It doesn't matter though, this is quite clear enough already :)

Reply:
I'm back from JS, sorry!

Reply:
Oh no, it was correct but @dacook's suggestion broke it. I was tempted to suggest the same thing but the query is not `> 0` but `> 1`. I thought that rails has a handy shortcut for this but couldn't find it when reviewing but I looked in the wrong place. An array has the `many?` method which is clearer. I'll add a suggestion.